client/src/theme.js

```diff
@@ -125,6 +125,15 @@ const theme = createTheme({
         },
       ],
     },
+    MuiCard: {
+      styleOverrides: {
+        root: {
+          padding: "2rem",
+          borderRadius: "2rem",
```

Comment:
Do we know the expected border radius for all the elements in this app? Did design create them uniform, or are there different classes of components that are using different border radius?

`theme.shapes.borderRadius`(double check) is a universal property that can control border radius for all of our components. Would be great not to have to tweak one thing at a time.

Reply:
the border radius is different for all components